src/style.css

```
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 30px 14px 15px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__icon {
+  margin: 0 5px 45px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 6px;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 15px;
+  color: #000;
+}
+
+.card__code {
+  margin-bottom: 15px;
+}
+
+.text {
+  font-size: 11px;
+  line-height: 13px;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__item {
+  margin-right: 4px;
+  width: 15px;
+  height: 15px;
+  background: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__item:nth-child(-n+4) {
+  background: url(../src/images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

if you use background, put repeat and position in one rule
```suggestion
  background: url(../src/images/star-active.svg) center no-repeat;
```